test/toml_test.rb

```diff
@@ -1,6 +1,46 @@
 require_relative 'helper'
 
 class TomlTest < Test::Unit::TestCase
+  def test_file_v_0_3_1
+    path = File.join(File.dirname(__FILE__), 'example-v0.3.0.toml')
+    parsed = TOML.load_file(path)
+
+    hash = {
+      "Table" => {
+        "key" => "value"
+      },
+      "dog" => {
+        "tater" => {
+          "type" => "pug"
+        }
+      },
+      "x" => {
+        "y" => {
+          "z" => {
+            "w" => {}
+          }
+        }
+      },
+     "String" => {
+        "basic" => "I'm a string. \"You can quote me\". Name\tJos\\u00E9\nLocation\tSF.",
```

Comment:
Line is too long. [89/80]<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.